packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -162,7 +175,11 @@ class TextPainter {
     if (_paragraph == null) {
       ui.ParagraphBuilder builder = new ui.ParagraphBuilder();
       _text.build(builder, textScaleFactor: textScaleFactor);
-      ui.ParagraphStyle paragraphStyle = _text.style?.getParagraphStyle(textAlign: textAlign, textScaleFactor: textScaleFactor);
+      ui.ParagraphStyle paragraphStyle = _text.style?.getParagraphStyle(
+          textAlign: textAlign,
```

Comment:
we usually do a two-character indent here. we'll probably change at some point to match typical dart style but for now let's keep the code consistent.